Doc/c-api/dict.rst

```diff
@@ -173,6 +173,23 @@ Dictionary Objects
 
    .. versionadded:: 3.4
 
+
+.. c:function:: int PyDict_Pop(PyObject *p, PyObject *key, PyObject *default_value, PyObject *result)
+
+   Remove *key* from dictionary *p* and return the removed value.
+
+   - If the key is present, set *\*result* to a :term:`strong reference` to the
```

Comment:
I'm curious: don't we use "new reference" usually?

Reply:
I changed the doc to use "new reference", instead of "strong reference".